actionpack/test/controller/parameters/accessors_test.rb

```diff
@@ -82,7 +82,7 @@ class ParametersAccessorsTest < ActiveSupport::TestCase
   end
 
   test "empty? returns false when any params are present" do
-    refute @params.empty?
+    assert_not @params.empty?
```

Comment:
since you are here use `assert_not_predicate`

Reply:
`assert_not_empty` seems even better for this case